packages/@react-aria/calendar/src/useCalendarBase.ts

```diff
@@ -65,6 +65,7 @@ export function useCalendarBase(props: CalendarPropsBase & DOMProps, state: Cale
       onPress: () => state.focusPreviousPage(),
       'aria-label': formatMessage('previous'),
       isDisabled: props.isDisabled || state.isPreviousVisibleRangeInvalid()
-    }
+    },
+    title: visibleRangeDescription
```

Comment:
Good name? Bad name? Is it clear enough? Would `visibleRangeDescription` be better, or too long? 🤔